spec/controllers/application_drafts_controller_spec.rb

```diff
@@ -0,0 +1,271 @@
+           number: @question.number,
+           direction: @direction
+    end
+    context 'student' do
+      before :each do
+        when_current_user_is :student
+      end
+      it 'does not allow access' do
+        expect_any_instance_of(ApplicationDraft)
+          .not_to receive(:move_question)
+        submit
+        expect(response).to have_http_status :unauthorized
+      end
+    end
+    context 'staff' do
+      before :each do
+        when_current_user_is :staff
+      end
+      it 'assigns the correct draft to the draft instance variable' do
+        submit
+        expect(assigns.fetch :draft).to eql @draft
+      end
+      it 'calls #move_question on the draft' do
+        expect_any_instance_of(ApplicationDraft)
+          .to receive(:move_question)
+          .with(@question.number, @direction)
+        submit
+      end
+      it 'redirects to the edit path' do
+        submit
+        expect(response).to redirect_to edit_draft_path(@draft)
+      end
+    end
+  end
+
+  describe 'POST #remove_question' do
+    before :each do
+      @draft = create :application_draft
+      @question = create :question, application_draft: @draft
+    end
+    let :submit do
+      post :remove_question, id: @draft.id, number: @question.number
+    end
+    context 'student' do
+      before :each do
+        when_current_user_is :student
+      end
+      it 'does not allow access' do
+        expect_any_instance_of(ApplicationDraft)
+          .not_to receive :remove_question
+        submit
+        expect(response).to have_http_status :unauthorized
+      end
+    end
+    context 'staff' do
+      before :each do
+        when_current_user_is :staff
+      end
+      it 'assigns the correct draft to the draft instance variable' do
+        submit
+        expect(assigns.fetch :draft).to eql @draft
+      end
+      it 'calls #remove_question on the draft with the number referenced' do
+        expect_any_instance_of(ApplicationDraft)
+          .to receive(:remove_question)
+          .with @question.number
+        submit
+      end
+      it 'redirects to the edit path for the draft' do
+        submit
+        expect(response).to redirect_to edit_draft_path(@draft)
+      end
+    end
+  end
+
+  describe 'POST #update' do
+    before :each do
+      @draft = create :application_draft
+    end
+    let :submit do
+      post :update, id: @draft
+    end
+    context 'student' do
+      before :each do
+        when_current_user_is :student
+      end
+      it 'does not allow access' do
+        expect_any_instance_of(ApplicationDraft)
+          .not_to receive :update
+        submit
+        expect(response).to have_http_status :unauthorized
+      end
+    end
+    context 'staff' do
+      before :each do
+        when_current_user_is :staff
+      end
+      it 'updates the draft with the changes given' do
+        expect { submit }
+        # TO WHAT? Change question count by one? how make that happen?
```

Comment:
Ah, I suspect this is the cause of that method being untested according to simplecov. I'm kind of surprised that `expect` without a `to` even passes.

I notice that your `submit` let block doesn't have any changes that should be made here. A good place to start would be not worrying about questions at all - just submit an update to an attribute of the draft, and make sure that this controller method does that change (I'm pretty sure this is all the corresponding test on ft-forms does).